library/core/src/ffi/c_str.rs

```diff
@@ -322,8 +319,9 @@ impl CStr {
     /// assert_eq!(c_str.to_str().unwrap(), "AAAAAAAA");
     /// ```
     ///
-    #[unstable(feature = "cstr_from_bytes_until_nul", issue = "95027")]
-    #[rustc_const_unstable(feature = "cstr_from_bytes_until_nul", issue = "95027")]
+    #[rustc_allow_const_fn_unstable(const_slice_index)]
+    #[stable(feature = "cstr_from_bytes_until_nul", since = "CURRENT_RUSTC_VERSION")]
+    #[rustc_const_stable(feature = "cstr_from_bytes_until_nul", since = "CURRENT_RUSTC_VERSION")]
```

Comment:
So this doesn't just stabilize const_cstr_methods but also cstr_from_bytes_until_nul?

Reply:
https://github.com/rust-lang/rust/pull/107429 stabilizes cstr_from_bytes_until_nul, I just branched off of that to avoid some redundant discussion about the changes to the memchr module (which both need).

But er... perhaps I've just made it a bit more confusing. I'll delete this bit from the PR contents when I get a chance.

Reply:
That one merged, so this tree is clean now